src/app/Routes/FaucetRoutes.tsx

```diff
@@ -17,7 +15,7 @@ const FaucetRoutes: any = () => {
 
   return (
     <Layout>
-      <AppBar excludeNetworks={[MAINNET_NETWORK_ID]} />
```

Comment:
make sure we don't have mainnet button on faucet app

Reply:
Now we choose the networks for each app in the Wagmi config

```typescript
const appChains: Record<typeof app, [Chain, ...Chain[]]> = {
  ssvweb: [mainnet, holesky],
  distribution: [mainnet],
  faucet: [holesky],
};
```